tool/resources/org/antlr/v4/tool/templates/codegen/Go/Go.stg

```diff
@@ -1014,7 +1014,7 @@ ListLabelName(label) ::= "<label>"
 CaptureNextToken(d) ::= "<d.varName> = p.GetTokenStream().LT(1)"
 CaptureNextTokenType(d) ::= "<d.varName> = p.GetTokenStream().LA(1)"
 
-StructDecl(struct, ctorAttrs, attrs, getters, dispatchMethods, interfaces, extensionMembers) ::= <<
+StructDecl(struct, ctorAttrs, attrs, getters, dispatchMethods, interfaces, extensionMembers, signatures) ::= <<
```

Comment:
Hmm I should probably also add this new `signatures` parameter to the `StructDecl` template in all the other targets as well, eh?

Reply:
(Added them)